output/list.go

```diff
@@ -32,31 +34,127 @@ import (
 var green = color.New(color.FgGreen).SprintFunc()
 var yellow = color.New(color.FgYellow).SprintFunc()
 var red = color.New(color.FgRed).SprintFunc()
+var blue = color.New(color.FgBlue).SprintFunc()
 
-type list struct{}
+// List is the output reporter as a list, with compact option.
+type List struct {
+	Compact bool
+	dotted  bool
+}
 
-// List is the output reporter as a list.
-var List list
+func (list *List) printf(format string, a ...interface{}) (n int, err error) {
+	if list.dotted {
+		list.dotted = false
+		fmt.Println()
+	}
+	return fmt.Printf(format, a...)
+}
 
-func (list) Start(plan *plan.Plan) error {
+func (list *List) printDot(status execute.Status) {
+	list.dotted = true
+	var dot string
+	switch status {
+	case execute.Skipped:
+		dot = yellow(".")
+	default:
+		dot = "."
+	}
+	fmt.Printf(dot)
+}
+
+func (list *List) Start(plan *plan.Plan) error {
 	return nil
 }
 
-func (list) Next(test execute.TestResponse) {
+func statusToColoredSymbol(status execute.Status) string {
+	switch status {
+	case execute.Success:
+		return green("✓")
```

Comment:
what do you mean by its not printing anything? are you talking about this: `return green("✓")`